lib/src/widgets/flutter_deck_code_highlight.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_highlight/flutter_highlight.dart';
+import 'package:flutter_highlight/themes/default.dart';
+import 'package:flutter_highlight/themes/vs2015.dart';
+
+/// This widget provides syntax highlighting for many languages.
+class FlutterDeckCodeHighlight extends StatelessWidget {
+  /// Creates a syntax highlighting widget for displaying code.
+  const FlutterDeckCodeHighlight({
```

Comment:
Can we also add the `TextStyle` property to the list? I feel that the current default `TextStyle` could be too small, would like to make it adjustable.

Reply:
Added `textStyle` to be merged with all the theme's text styles (like for changing the overall font size) and `darkTextStyles` and `lightTextStyles` to override the used theme completely.

Reply:
Are you sure it's needed to be that complex? It seems that it's enough to pass the `textStyle` to the `HighlightView` and these will be merged under the hood: https://github.com/git-touch/highlight.dart/blob/master/flutter_highlight/lib/flutter_highlight.dart#L86-L88

Reply:
Then you can only modify the `root` text style in the theme. If that is enough for you, I'm totally fine with removing the optional ability to customize everything else. It works fine if you only want to customize the font size for example.

Reply:
No, I think this is just enough for the use case, thanks!